handlers.go

```diff
@@ -119,7 +119,7 @@ func (s *Server) ServeAPI(srv string, logging bool, lsteptok string, cachedir st
 
 	e.GET("/test-error", s.handleTestError)
 
-	e.GET("/viewer", withUser(s.handleGetViewer), s.AuthRequired(util.PermLevelUser))
```

Comment:
I don't know if I agree with this but I think it allows for a full client flow so thats okay. Its just that the viewer might have information that the upload key shouldn't know about, but its fine for now since I don't think this is something that will live forever.

Reply:
Yeah, the main thing is that when youre uploading you need access to the set of shuttles to upload to, which requires hitting this endpoint.